clusters/app.ci/supplemental-ci-images/base-ci-images/images-ocp.yaml

```diff
@@ -69,6 +69,14 @@ spec:
       scheduled: false
     referencePolicy:
       type: Local
+  - name: "8.5"
+    from:
+      kind: DockerImage
+      name: registry.access.redhat.com/ubi8/ubi-minimal:8.5
+    importPolicy:
+      scheduled: false
+    referencePolicy:
+      type: Local
```

Comment:
Is there a doc that instructed you to change these files? Because these are not the right files to update, https://github.com/openshift/release/blob/master/core-services/image-mirroring/supplemental-ci-images/mapping_supplemental_ci_images_ocp ones are.

Reply:
Great, thanks for pointing out the right place to make this change.

The intention of my change is to use it here: https://github.com/openshift/release/pull/27377